prepare_mkl.sh

```diff
@@ -68,8 +68,8 @@ if [ ! -d "$HOME_MKL" ]; then
 fi
 MXNET_ROOT=`dirname $0`
 OMP=0 
-VERSION_MATCH=20161123
-ARCHIVE_BASENAME=mklml_lnx_2017.0.2.20161122.tgz
+VERSION_MATCH=20170210
+ARCHIVE_BASENAME=mklml_lnx_2017.0.2.20170209.tgz
```

Comment:
no .
MKLML_ROOT=/usr/local is fixed, difference mkl version will use the same MKLML_ROOT